prac_08/KivyDemos-master/box_layout_demo.py

```diff
@@ -0,0 +1,24 @@
+# Modify Existing GUI Program
+
+from kivy.app import App
+from kivy.lang import Builder
+
+
+class BoxLayoutDemo(App):
+    def build(self):
+        self.title = "Box Layout Demo"
+        self.root = Builder.load_file('box_layout.kv')
+        return self.root
+
+    def handle_greet(self):
+        print('greet')
+        self.root.ids.output_label.text = f"Hello {self.root.ids.input_name.text}"
+
+    def press_clear(self):
+        self.root.ids.output_label.text = ""
+        self.root.ids.input_name.text = ""
+
+
+BoxLayoutDemo().run()
+
+
```

Comment:
Missing docstring

Reply:
Point well taken, I will add docstrings for each function. Thank you for the review!